src/flash/display/DisplayObjectContainer.ts

```diff
@@ -94,6 +94,7 @@ module Shumway.AVM2.AS.flash.display {
           child.dispatchEvent(events.Event.getInstance(events.Event.AVM1_INIT));
           child.dispatchEvent(events.Event.getInstance(events.Event.AVM1_CONSTRUCT));
           child._setFlags(DisplayObjectFlags.NeedsLoadEvent);
+          this._setFlags(DisplayObjectFlags.ContainsFrameScriptPendingChildren);
```

Comment:
Shouldn't this be set conditionaly, only if HasFrameScriptPending is set on child?